envfile.sample

```diff
@@ -39,6 +38,8 @@ UAZ_HITS=10
 UVICORN_RELOAD=True
 #DATASET_STORAGE_BASE_URL=s3://jataware-world-modelers-dev/datasets/
 #DOCUMENT_STORAGE_BASE_URL=s3://jataware-world-modelers-dev/documents/
-DOCUMENT_STORAGE_BASE_URL=file:///documents/
+CONFIG_STORAGE_BASE=s3://dojo/configs/
```

Comment:
@mattprintz same for these values- are these Ok defaults for dev/prod? `file:///` was giving me issues today.

Reply:
Also `CONFIG_STORAGE_BASE` was missing from envfile.sample.

@Sorrento110 what do you think of these changes?